medusa/providers/generic_provider.py

```diff
@@ -493,15 +493,20 @@ def parse_pubdate(pubdate, human_time=False, timezone=None):
 
         :returns: a datetime object or None
         """
+        now_alias = ['right now', 'just now', 'Now']
```

Comment:
In the other branch where I did a more extensive refactor, I made the list extensible through param. I kind of copied the code, but didn't copy the extensibility. But in this case the list doesn't need to be mutable. So I can change that with the benefit of a small performance increase.

But to challenge that, I don't think there is really allot of performance increase in comparing a list of three strings.